packages/components/stories/icon/icons.js

```diff
@@ -1107,6 +1113,8 @@ export const icons =
         'IcWalletSuccessMessage',
         'IcWalletTetherDark',
         'IcWalletTetherLight',
+        'IcWalletUpgradeCompleted',
+        'IcWalletUpgradeLogout',
         'IcWalletUsdCoinDark',
         'IcWalletUsdCoinLight',
     ],
```

Comment:
nit: if im not mistaken, we don't need to push this file

Reply:
seems for new icons, this file is also needed.